tests/unit/Cassandra/TupleTest.php

```diff
@@ -137,24 +135,19 @@ public function compositeTypes()
 
     public function testValidatesTypesOfElements()
     {
-        $this->expectException(InvalidArgumentException::class);
-        $this->expectExceptionMessage(
-            "argument must be an instance of Cassandra\Varint, an instance of Cassandra\Decimal given"
-        );
-        $tuple = new Tuple(array(Cassandra::TYPE_VARINT));
+        $this->expectException(\InvalidArgumentException::class);
+        $this->expectExceptionMessage("argument must be an instance of Cassandra\Varint, an instance of Cassandra\Decimal given");
+        $tuple = new Tuple(array(\Cassandra::TYPE_VARINT));
         $tuple->set(0, new Decimal('123'));
     }
 
     public function testSetAllElements()
     {
-        $tuple = new Tuple(
-            array(
-                Cassandra::TYPE_BOOLEAN,
-                Cassandra::TYPE_INT,
-                Cassandra::TYPE_BIGINT,
-                Cassandra::TYPE_TEXT,
-            )
-        );
+        $tuple = new Tuple(array(\Cassandra::TYPE_BOOLEAN,
```

Comment:
Why did you try to indent them all? This is an example of a bad indent you introduced

Reply:
https://github.com/rgsystemes/php-driver/pull/2/commits/a9b5ba732ec751715cd84c0c52a38e71ce303ed9
<img width="1400" alt="image" src="https://user-images.githubusercontent.com/717869/158137955-3a57331b-caa7-4ca9-9947-84739dac4ed8.png">

https://github.com/rgsystemes/php-driver/pull/2/commits/48ee3a94f8a8f0c25ff2c9ed29adbff07aed9759
<img width="1388" alt="image" src="https://user-images.githubusercontent.com/717869/158138069-55288c70-3864-403e-b798-596ff73db32d.png">

I didn't badly indented, I reverted the nano-interactive indentation